classes/classes/Output.as

```diff
@@ -12,7 +12,17 @@ package classes
 		import classes.GlobalFlags.kFLAGS;
 		import coc.view.MainView;
 
-		public function Output() {}
+		private static var _instance:Output = new Output();
+
+		public function Output()
```

Comment:
Actually, in my PHP framework singletons have private and empty constructors.
But in FlashDevelop its very helpful when debuggin, since you'll get a nice backtrace, when you throw an Error. Hence I preferred that approach. The result is the same anyway.